dare-jump/src/App.js

```diff
@@ -1,26 +1,32 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect, createRef } from 'react'
+import styles from './App.module.css'
+import './App.css'
+
+const PLAYER_WIDTH = 32
+const STEP = 10
 
 function App() {
+  const [posX, setPosX] = useState(0)
+  const containerRef = createRef(null)
+  useEffect(() => {
+    const eventListener = event => {
+      const screenWidth = containerRef.current.clientWidth
+      if (event.key === 'ArrowRight' && posX < screenWidth - PLAYER_WIDTH) {
+        setPosX(posX + STEP)
```

Comment:
I would add another condition here, if the remaining space is smaller than STEP then move the remaining space instead.
This would avoid going outside the window and creating overflow when the screen's width is not multiple of STEP.

Reply:
very good catch 👍